### PR TITLE
Enable in code definition of paths

### PR DESCRIPTION
**Pull request by abraovic**
Hi,

First let me thank you for sharing this project. It is something that's really useful when you have multi-dev team or when deploying stuff to a server.

I've added some changes that made me use this lib in a more flexible way. The thing was I couldn't use relative paths since is_readable php method doesn't support them. I believe that file_exists is suitable replacement since most of the files that are created are readable, and if not all you have to do is make them.

One other thing I've added is an option to set paths in code. This enables to use CLI command in a more clean fashion, while setting all paths in code.

If you agree on this please add it to a library.

Best,
Ante.

TLDR;
Changes:
- changed default paths to be public static instead of const so that those can be set inside the recipe
- replaced is_readable with file_exists because is_readable doesn't support relative paths

### CODE REVIEW

**Comment by eXistenZNL** [comment]
This project has not been maintained for years, so closing this pull request.
